### PR TITLE
changes according to the code reviewer

### PR DESCRIPTION
**Pull request by Rifat8080**
- Added all request changes according to the previous reviewer
- Added live demo link
- Fixed all margins and paddings

### CODE REVIEW

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Rifat8080  ✋

Good job so far!

## Highlights!
- Followed GitHub Flow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

## I would recommend you maintain the same pull request until it is approved to help code reviewers perform quality reviews.

# Status: Required Changes ♻️⛔️

- [x] As given in the project guidelines kindly ensure your design is a replica of the provided design. You may align the elements of your pages correctly so as to replicate the design given.
- Your design
your desktop-1
your-desktop-2
- Given design
Given desktop

- [ ] Kindly ensure your programs section has five cards instead of three so that your design can be an imitation of the design given

- Your design
your programs
- Given design
Given programs

- [x] Kindly remove this button from the desktop view. It only exists on the mobile view of the home section.
- Your design
your desktop button

- Given design
given desktop button

- [ ] Kindly ensure the footer section of your about is a replica of the design given.
- Your design
Your-about section footer
- Given design
given about footer


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
```
Comment (Shedrack-Sunday) [inline about.css:5]: - [ ] As suggested by the last reviewer, kindly add two more cards to have the same number as specified in the design given.
-The idea is to have you replicate the same design. Kindly add the remaining cards 👍 
- [Optional] The color of the cards is not the same as yours. To have a better design and have a replica that fits perfectly, kindly consider adding the right color which is grey, and The hover on the cards which the last reviewer missed 👍 

## Your work:
image

## Design:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Rifat8080        👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use of GitHub features ✔️   
- Good job heeding most of the recommendations of the last reviewer  👍 

  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few  features you missed or need to implement

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Ibizugbe — APPROVED** [review]
Hi @Rifat8080 👋👋👋

VeryNiceNiceGIF (2)
, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅

- [x] linters are passing👍.
- [x] The appropriate number of cards have been added as requested by the previous reviewer👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**